gix-transport/src/client/blocking_io/http/mod.rs

```diff
@@ -179,6 +179,8 @@ pub struct Options {
     pub ssl_ca_info: Option<PathBuf>,
     /// The SSL version or version range to use, or `None` to let the TLS backend determine which versions are acceptable.
     pub ssl_version: Option<SslVersionRangeInclusive>,
+    /// Controls whether to perform ssl identity verification or not
```

Comment:
Can you explain more about this? Why would one want to turn that on, or off. Are there security implications?

All that can be written in the doc-string. Please write in full sentences as well, which end in a `.` .

Reply:
I added more explanation.